data_structures/src/chain.rs

```diff
@@ -163,8 +200,25 @@ pub struct BlockHeader {
     pub version: u32,
     /// A checkpoint beacon for the epoch that this block is closing
     pub beacon: CheckpointBeacon,
-    /// A 256-bit hash based on all of the transactions committed to this block
-    pub hash_merkle_root: Hash,
+    /// A 256-bit hashes based on all of the transactions committed to this block
+    pub merkle_roots: BlockMerkleRoots,
+}
+/// Block merkle tree roots
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, ProtobufConvert, Default)]
+#[protobuf_convert(pb = "witnet::Block_BlockHeader_BlockMerkleRoots")]
+pub struct BlockMerkleRoots {
+    /// A 256-bit hash based on the mint transaction committed to this block
+    pub mint_hash_merkle_root: Hash,
```

Comment:
This is interesting, but I'm undecided if strictly necessary. In any case, it's not a merkle root but a plain hash, so we should name it accordingly :wink: